modules/imaging_browser/templates/imaging_session_controlpanel.tpl

```diff
@@ -1,9 +1,9 @@
-    <h3>Navigation</h3>
+<h3>{dgettext("imaging_browser","Navigation")}</h3>
```

Comment:
Same with this file:
<img width="214" height="713" alt="image" src="https://github.com/user-attachments/assets/e58f7563-036a-475e-88d8-cf6988cae35a" />